src/test/java/net/minecraftforge/debug/entity/living/VillagerChangeProfessionEventTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.entity.living;
+
+import com.mojang.logging.LogUtils;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.living.VillagerEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.slf4j.Logger;
+
+@Mod("villager_change_profession_event_test")
+public class VillagerChangeProfessionEventTest {
```

Comment:
Please provide a javadoc for this class which explains what feature this tests, how to test it, and what the expected results are from said test.